specs/shell/search_files_and_process.yaml

```diff
@@ -0,0 +1,20 @@
+---
+name: Search for specified file types and process the file
```

Comment:
Just to confirm, by `process` here, you mean run a certain command for each file? If so, could we make that more clear in the description?

Reply:
Thank you, @alokedesai! Just updated the description as you suggested.